identity-composer-component.html

```diff
@@ -1,31 +1,137 @@
+<style>
```

Comment:
All of these special classes should be prefixed as well with `br-` -- CSS classes are global.